.github/actions/sourceclear/Dockerfile

```
@@ -1,4 +1,5 @@
 FROM node:16-alpine
 RUN apk add --no-cache jq
+RUN apk --no-cache add curl
```

```suggestion
FROM node:16
RUN apt-get install -y --no-install-recommends curl jq
```
node:16 image is based on https://github.com/docker-library/buildpack-deps/blob/65d69325ad741cea6dee20781c1faaab2e003d87/debian/buster/scm/Dockerfile